bodo/tests/test_df_lib/test_jit_interop.py

```diff
@@ -14,6 +16,24 @@ def test_bodo_df_to_jit():
         }
     )
     bdf = bd.from_pandas(df)
+    return df, bdf
+
+
+def df_from_parquet():
+    path = datapath_util("example.parquet")
```

Comment:
```suggestion
    path = datapath_util("example_no_index.parquet")
```
This test will fail now since there is an index column, was silently passing before since we just ignored the index and it happened to match.